janitor/timeseries.py

```diff
@@ -24,31 +24,40 @@ def fill_missing_timestamps(
     If timestamps are not missing, then the function will return the DataFrame
     unmodified.
 
-    Functional usage example:
-
-    ```python
-    import pandas as pd
-    import janitor.timeseries
-
-    df = pd.DataFrame(...)
-
-    df = janitor.timeseries.fill_missing_timestamps(
-        df=df,
-        frequency="1H",
-    )
-    ```
-
     Method chaining example:
 
-    ```python
-    import pandas as pd
-    import janitor.timeseries
+    >>> import pandas as pd
+    >>> import janitor.timeseries
+    >>> from random import randint
+
+    >>> ts_index = pd.date_range("1/1/2019", periods=4, freq="1H")
+    >>> v1 = [randint(1, 2000) for i in range(4)]
+    >>> df = (pd.DataFrame(pd.DataFrame({"v1": v1}, index=ts_index)
+    ...         .fill_missing_timestamps(frequency="1H") )
+    ...      )
+    >>> df
+                           v1
+    2019-01-01 00:00:00  418
+    2019-01-01 01:00:00  1610
+    2019-01-01 02:00:00  339
+    2019-01-01 03:00:00  1458
+    >>> df.drop(df.index[1])
+                           v1
+    2019-01-01 00:00:00   418
+    2019-01-01 02:00:00   339
+    2019-01-01 03:00:00  1458
```

Comment:
1. Please change the section header to "Examples:" instead of "Method chaining example:".
2. You'ld need to indent the entire example block (i.e., the chevrons `>>>` and `...` by 4 more spaces). Otherwise the result looks like this: 
![image](https://user-images.githubusercontent.com/30731072/167280379-ec04b7a1-6a0f-421e-b7a6-186de7660f06.png)

Tip: you may run `mkdocs serve` in the project root to see what the documentation will look like.